### PR TITLE
[Merged by Bors] - Add mouse grab example

### LINKED ISSUE #4094: Add example for grabbing the mouse
## How can Bevy's documentation be improved?

Demonstrate [how to grab the mouse / lock the cursor] in a small example.

### PR DESCRIPTION
**Pull request by emersonmx**
Fixes #4094

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -203,6 +203,7 @@ Example | File | Description
 `keyboard_modifiers` | [`input/keyboard_modifiers.rs`](./input/keyboard_modifiers.rs) | Demonstrates using key modifiers (ctrl, shift)
 `mouse_input` | [`input/mouse_input.rs`](./input/mouse_input.rs) | Demonstrates handling a mouse button press/release
 `mouse_input_events` | [`input/mouse_input_events.rs`](./input/mouse_input_events.rs) | Prints out all mouse events (buttons, movement, etc.)
+`mouse_grab` | [`input/mouse_grab.rs`](./input/mouse_grab.rs) | Demonstrates how to grab the mouse
```
Comment (alice-i-cecile) [inline examples/README.md:206]:
```suggestion
`mouse_grab` | [`input/mouse_grab.rs`](./input/mouse_grab.rs) | Demonstrates how to grab the mouse, locking the cursor to the app's screen
```

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Looks good! Just a tiny suggestion to improve the description.

**Review by alice-i-cecile — APPROVED** [review]

File: examples/input/mouse_grab.rs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_system(grab_mouse)
+        .run();
+}
+
+// This system grab the mouse when you press the left mouse button and release
```
Comment (james7132) [inline examples/input/mouse_grab.rs:10]:
```suggestion
// This system grabs the mouse when you the left mouse button is pressed and releases
```

File: examples/input/mouse_grab.rs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+// when press escape key:
```
Comment (james7132) [inline examples/input/mouse_grab.rs:11]:
```suggestion
// when the escape key is pressed:
```

**Review by james7132 — CHANGES_REQUESTED** [review]

File: examples/input/mouse_grab.rs (hunk 2 above)
Comment (alice-i-cecile) [reply examples/input/mouse_grab.rs:10]:
```suggestion
// This system grabs the mouse when the left mouse button is pressed
// and releases it when the escape key is pressed
```

IMO splitting this across two lines to match the sentence structure is clearer.

**Comment by alice-i-cecile** [comment]
Awesome. Once I have a second approval I'm ready to merge this :)

**Review by james7132 — APPROVED** [review]

**Pull request closed without merging**